docs/end-user/components/cloud-services/prepare-jdbc-connection-string.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Prepare JDBC connection string
+---
+
+It's common to use the JDBC connection string to connect to a database. This doc will focus on how to prepare the JDBC connection string
+and how to use it in the application.
+
+## How to generate the JDBC connection string for business component
+
+- Apply WorkflowStep `generate-jdbc-connection`
+
+> WorkflowStep `generate-jdbc-connection` will be automatically applied with KubeVela chart installation. It's not necessary
+> to apply it manually when this feature is released.
+
+```shell
+$ vela def apply generate-jdbc-connection.cue
+```
+
+```
+# generate-jdbc-connection.cue
+import (
+	"vela/op"
+	"encoding/base64"
+)
+
+"generate-jdbc-connection": {
+	type: "workflow-step"
+	annotations: {}
+	labels: {
+		"ui-hidden": "true"
+	}
+	description: "Generate a JDBC connection based on Component of alibaba-rds"
+}
+template: {
+	output: op.#Read & {
+		value: {
+			apiVersion: "v1"
+			kind:       "Secret"
+			metadata: {
+				name: parameter.name
+				if parameter.namespace != _|_ {
+					namespace: parameter.namespace
+				}
+			}
+		}
+	}
+	dbHost:   op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_HOST"])}
+	dbPort:   op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_PORT"])}
+	dbName:   op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_NAME"])}
+	username: op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_USER"])}
+	password: op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_PASSWORD"])}
+
+	env: [
+		{name: "url", value:      "jdbc://" + dbHost.str + ":" + dbPort.str + "/" + dbName.str + "?characterEncoding=utf8&useSSL=false"},
+		{name: "username", value: username.str},
+		{name: "password", value: password.str},
+	]
+
+	parameter: {
+		// +usage=Specify the name of the secret generated by database component
+		name: string
+		// +usage=Specify the namespace of the secret generated by database component
+		namespace?: string
+	}
+}
+
+```
+
+The property `namespace` and `name` refers to the namespace and name of the secret generated by a database component.
+
+```
+$ vela show generate-jdbc-connection
+# Properties
++-----------+---------------------------------------------------------------------+--------+----------+---------+
+|   NAME    |                             DESCRIPTION                             |  TYPE  | REQUIRED | DEFAULT |
++-----------+---------------------------------------------------------------------+--------+----------+---------+
+| name      | Specify the name of the secret generated by database component      | string | true     |         |
+| namespace | Specify the namespace of the secret generated by database component | string | false    |         |
++-----------+---------------------------------------------------------------------+--------+----------+---------+
+```
+
+[The keys of the secret]((./terraform/alibaba-rds#outputs)) are as below:
```

Comment:
I think you need to use single brackets to avoid broken links.

Reply:
Thanks:)